css/style.css

```diff
@@ -584,6 +596,7 @@ ul {
 .about-img {
   border-radius: 20px;
   max-width: 100%;
+  margin-right: 15px;
```

Comment:
margin: 0 15px; - on both sides (left and right) for mobile, and then from tablet you can set it back to margin:  0; because we won't need it then